package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@intacct-thyme/intacct-sdk",
-  "version": "2.2.0",
```

Comment:
This is a (minor) bug on latest. `npm i` needs to be run when bumping `package.json` when using NPM's `package-lock.json`.